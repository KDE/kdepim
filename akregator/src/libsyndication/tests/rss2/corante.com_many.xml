<?xml version="1.0" encoding="UTF-8"?>
<!--
    properties:
      - movable type example feed
      - uses feedburner vocab
      - <author> has <name> tag inside
      -->
<?xml-stylesheet href="http://feeds.feedburner.com/~d/styles/rss2full.xsl" type="text/xsl" media="screen"?><?xml-stylesheet href="http://feeds.feedburner.com/~d/styles/itemcontent.css" type="text/css" media="screen"?><rss xmlns:feedburner="http://rssnamespace.org/feedburner/ext/1.0" version="2.0">
<channel>
<title>Many-to-Many</title>
<link>http://many.corante.com/</link>
<description />
<language>en-us</language>
<copyright>Copyright 2006</copyright>
<lastBuildDate>Mon, 13 Mar 2006 15:58:36 -0500</lastBuildDate>
<generator>http://www.movabletype.org/?v=3.2</generator>
<docs>http://blogs.law.harvard.edu/tech/rss</docs> 

<atom10:link xmlns:atom10="http://www.w3.org/2005/Atom" rel="self" href="http://feeds.feedburner.com/Many-to-many" type="application/rss+xml" /><item>
<title>glocalization talk at Etech</title>
<description>&lt;p&gt;Last week, i gave a talk at &lt;span class="caps"&gt;O&amp;#8217;R&lt;/span&gt;eilly&amp;#8217;s Etech on how large-scale digital communities can handle the tensions between global information networks and local interaction and culture.   I&amp;#8217;ve uploaded the crib for those who are interested in reading the talk: &lt;a href="http://www.danah.org/papers/Etech2006.html"&gt;&amp;#8220;G/localization: When Global Information and Local Interaction Collide&amp;#8221;&lt;/a&gt;. &lt;/p&gt;

&lt;p&gt;This talk was written for designers and business folks working in social tech.  I talk about the significance of culture and its role in online communities.  I go through some of the successful qualities of Craiglist, Flickr and MySpace to lay out a critical practice: design through embedded observation.  I then discuss a few issues that are playing out on tech and social levels.&lt;/p&gt;

&lt;p&gt;Anyhow, enjoy! And let me know what you think!&lt;/p&gt;
&lt;p&gt;&lt;a href="http://feeds.feedburner.com/~a/Many-to-many?a=rlK0jP"&gt;&lt;img src="http://feeds.feedburner.com/~a/Many-to-many?i=rlK0jP" border="0"&gt;&lt;/img&gt;&lt;/a&gt;&lt;/p&gt;&lt;img src="http://feeds.feedburner.com/Many-to-many?g=23"/&gt;</description>

<link>http://feeds.feedburner.com/Many-to-many?m=23</link>
<guid isPermaLink="false">http://many.corante.com/archives/2006/03/13/glocalization_talk_at_etech.php</guid>
<author><name>danah</name></author>
<category>social software</category>
<pubDate>Mon, 13 Mar 2006 15:58:36 -0500</pubDate>
<feedburner:awareness>http://api.feedburner.com/awareness/1.0/GetItemData?uri=Many-to-many&amp;itemurl=http%3A%2F%2Fmany.corante.com%2Farchives%2F2006%2F03%2F13%2Fglocalization_talk_at_etech.php</feedburner:awareness><feedburner:origLink>http://many.corante.com/archives/2006/03/13/glocalization_talk_at_etech.php</feedburner:origLink></item>
<item>
<title>Clash of Uncivilizations</title>
<description>&lt;p&gt;&lt;a href="http://www.minorityrapport.com/"&gt;Jon Turow&lt;/a&gt; passed on an &lt;a href="http://www.dailyprincetonian.com/archives/2006/03/09/arts/14810.shtml"&gt;open letter&lt;/a&gt; to Mark Zuckerberg in the Daily Princetonian.  Facebook recently expanded from college to &lt;a href="http://hs.facebook.com/"&gt;high school&lt;/a&gt;, resulting in a clash of uncivilizations: &lt;/p&gt;

&lt;blockquote&gt;&amp;#8230;If we really wanted to, we could steer clear of the groups by just avoiding the high school profiles. But we can&amp;#8217;t ignore it when they post on our walls. And my god, do they post. Unfortunately, they don&amp;#8217;t understand that by posting &amp;#8220;OMG how are you? I haven&amp;#8217;t seen you since our Model UN trip three years ago!&amp;#8221; they are undermining the college personas that we have so carefully constructed over the past there years. And when a 16-year-old girl pokes us, we worry that poking back could result in a cyber-statutory rape conviction. Something tells us that when having sex with one of your facebook friends could result in a criminal violation, things have gone too far&amp;#8230;.&lt;/blockquote&gt;
&lt;p&gt;&lt;a href="http://feeds.feedburner.com/~a/Many-to-many?a=MT4U3V"&gt;&lt;img src="http://feeds.feedburner.com/~a/Many-to-many?i=MT4U3V" border="0"&gt;&lt;/img&gt;&lt;/a&gt;&lt;/p&gt;&lt;img src="http://feeds.feedburner.com/Many-to-many?g=22"/&gt;</description>

<link>http://feeds.feedburner.com/Many-to-many?m=22</link>
<guid isPermaLink="false">http://many.corante.com/archives/2006/03/12/clash_of_uncivilizations.php</guid>
<author><name>Ross</name></author>
<category>social software</category>
<pubDate>Sun, 12 Mar 2006 10:45:25 -0500</pubDate>
<feedburner:awareness>http://api.feedburner.com/awareness/1.0/GetItemData?uri=Many-to-many&amp;itemurl=http%3A%2F%2Fmany.corante.com%2Farchives%2F2006%2F03%2F12%2Fclash_of_uncivilizations.php</feedburner:awareness><feedburner:origLink>http://many.corante.com/archives/2006/03/12/clash_of_uncivilizations.php</feedburner:origLink></item>



<feedburner:awareness>http://api.feedburner.com/awareness/1.0/GetFeedData?uri=Many-to-many</feedburner:awareness></channel>
</rss>
