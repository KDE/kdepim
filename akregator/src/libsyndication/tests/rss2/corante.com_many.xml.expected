# Feed begin ######################
title: #Many-to-Many#
link: #http://many.corante.com/#
description: ##
copyright: #Copyright 2006#
language: #en-us#
# Item begin ######################
id: #http://many.corante.com/archives/2006/03/13/glocalization_talk_at_etech.php#
title: #glocalization talk at Etech#
link: #http://feeds.feedburner.com/Many-to-many?m=23#
description: #<p>Last week, i gave a talk at <span class="caps">O&#8217;R</span>eilly&#8217;s Etech on how large-scale digital communities can handle the tensions between global information networks and local interaction and culture. I&#8217;ve uploaded the crib for those who are interested in reading the talk: <a href="http://www.danah.org/papers/Etech2006.html">&#8220;G/localization: When Global Information and Local Interaction Collide&#8221;</a>. </p> <p>This talk was written for designers and business folks working in social tech. I talk about the significance of culture and its role in online communities. I go through some of the successful qualities of Craiglist, Flickr and MySpace to lay out a critical practice: design through embedded observation. I then discuss a few issues that are playing out on tech and social levels.</p> <p>Anyhow, enjoy! And let me know what you think!</p> <p><a href="http://feeds.feedburner.com/~a/Many-to-many?a=rlK0jP"><img src="http://feeds.feedburner.com/~a/Many-to-many?i=rlK0jP" border="0"></img></a></p><img src="http://feeds.feedburner.com/Many-to-many?g=23"/>#
datePublished: #Mon Mar 13 21:58:36 2006#
dateUpdated: #Mon Mar 13 21:58:36 2006#
# Person begin ####################
name: #danah#
# Person end ######################
# Category begin ##################
term: #social software#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #http://many.corante.com/archives/2006/03/12/clash_of_uncivilizations.php#
title: #Clash of Uncivilizations#
link: #http://feeds.feedburner.com/Many-to-many?m=22#
description: #<p><a href="http://www.minorityrapport.com/">Jon Turow</a> passed on an <a href="http://www.dailyprincetonian.com/archives/2006/03/09/arts/14810.shtml">open letter</a> to Mark Zuckerberg in the Daily Princetonian. Facebook recently expanded from college to <a href="http://hs.facebook.com/">high school</a>, resulting in a clash of uncivilizations: </p> <blockquote>&#8230;If we really wanted to, we could steer clear of the groups by just avoiding the high school profiles. But we can&#8217;t ignore it when they post on our walls. And my god, do they post. Unfortunately, they don&#8217;t understand that by posting &#8220;OMG how are you? I haven&#8217;t seen you since our Model UN trip three years ago!&#8221; they are undermining the college personas that we have so carefully constructed over the past there years. And when a 16-year-old girl pokes us, we worry that poking back could result in a cyber-statutory rape conviction. Something tells us that when having sex with one of your facebook friends could result in a criminal violation, things have gone too far&#8230;.</blockquote> <p><a href="http://feeds.feedburner.com/~a/Many-to-many?a=MT4U3V"><img src="http://feeds.feedburner.com/~a/Many-to-many?i=MT4U3V" border="0"></img></a></p><img src="http://feeds.feedburner.com/Many-to-many?g=22"/>#
datePublished: #Sun Mar 12 16:45:25 2006#
dateUpdated: #Sun Mar 12 16:45:25 2006#
# Person begin ####################
name: #Ross#
# Person end ######################
# Category begin ##################
term: #social software#
# Category end ####################
# Item end ########################
# Feed end ########################

